src/components/Radio/stories.js

```diff
@@ -1,46 +1,178 @@
 import Radio from ".";
 
+const onSubmit = values => {
+  console.log(JSON.stringify(values, 0, 2));
+};
+
+const required = value => (value ? undefined : "Required");
+
+const RadioGroup = ({ label, hint, name, options, inline, ...input }) => (
+  <Layout>
+    <GridRow>
+      <GridCol>
+        <MultiChoice {...input} label={label} hint={hint}>
+          {options.map(o => (
+            <div key={o.value}>
+              <Radio
+                type="radio"
+                {...input}
+                inline={inline}
+                value={o.value}
+                checked={o.value === input.value}
+              >
+                {o.title}
+              </Radio>
+            </div>
+          ))}
+        </MultiChoice>
+      </GridCol>
+    </GridRow>
+  </Layout>
+);
+
+RadioGroup.defaultProps = {
+  hint: undefined,
+  inline: false,
+  name: undefined,
+  options: {}
+};
+
+RadioGroup.propTypes = {
+  label: PropTypes.string.isRequired,
+  hint: PropTypes.string,
+  name: PropTypes.string,
+  inline: PropTypes.bool,
+  options: PropTypes.shape({
+    title: PropTypes.string,
+    value: PropTypes.string
+  })
+};
+
+const App = () => (
+  <div>
+    <Form
+      onSubmit={onSubmit}
+      render={({ handleSubmit, reset, submitting, pristine, values }) => (
+        <form onSubmit={handleSubmit}>
+          <Layout>
+            <GridRow>
+              <GridCol>
+                <Field
+                  name="likesAnimals"
+                  label="Do you like animals?"
+                  hint="You must tell us"
+                  component={RadioGroup}
+                  type="radio"
+                  options={[
+                    { title: "Yep", value: "yes" },
+                    { title: "Nope", value: "no" }
+                  ]}
+                  validate={required}
+                  inline
+                />
+                {/* <label>
+                  <Field
+                    type="radio"
+                    component="input"
+                    value="yes"
+                    name="test"
+                  />
+                  Yes
+                </label>
+                <label>
+                  <Field
+                    type="radio"
+                    component="input"
+                    value="no"
+                    name="test"
+                  />
+                  No
+                </label> */}
+              </GridCol>
+            </GridRow>
+            <GridRow>
+              <GridCol>
+                <Button type="submit" disabled={submitting}>
+                  Log In
+                </Button>
+              </GridCol>
+              <GridCol>
+                <Button onClick={reset} disabled={submitting || pristine}>
+                  Reset
+                </Button>
+              </GridCol>
+            </GridRow>
+            <GridRow>
+              <GridCol>
+                <hr />
+                <pre>{JSON.stringify(values, 0, 2)}</pre>
```

Comment:
may be able to do something nice with `@storybook/addon-actions` instead of showing inline